ModConfigMenu/ModConfigMenu/Src/ModConfigMenuAPI/Classes/MCM_API_Instance.uc

```diff
@@ -1,5 +1,13 @@
 interface MCM_API_Instance;
 
+enum eSettingEventMode
+{
+    // Callbacks are triggered when the user manipulates the UI element. Use SetSaveHandler and SetCancelHandler to save/abort.
+    eSettingEventMode_Immediate,

```

Comment:
Each control has its own `OnChanged` callback, and the page has the `OnSave` callback.  See #13